src/components/inscripcionTurnos/InscripcionActividad.js

```diff
@@ -31,6 +32,10 @@ export default function Actividad() {
   const { create } = useApi('turnos');
   const usuario = useRecoilValue(usuarioState);
   const [iconoCargando, setIconoCargando] = useState(false);
+  const [errorAlPedirTurno, setErrorAlPedirTurno] = useState({
+    hayError: false,
+    mensaje: '',
+  });
```

Comment:
Es redundante y propenso a errores el booleano, basta con que el state sea un string.


```suggestion
  const [errorAlPedirTurno, setErrorAlPedirTurno] = useState();
```